src/selection-recommendation/test_class.py

```diff
@@ -0,0 +1,33 @@
+import time_conflicts_check
+
```

Comment:
I will create a directory called tests under the parent directory, can you move this file to that place?